homeassistant/components/velux/cover.py

```diff
@@ -46,6 +46,11 @@ async def async_added_to_hass(self):
         """Store register state change callback."""
         self.async_register_callbacks()
 
+    @property
+    def unique_id(self):
+        """Return the unique ID of this cover."""
+        return f"{DOMAIN}-{self.node.pyvlx.config.host}-{self.node.node_id}"
```

Comment:
The host is not a persistent unique ID.

A unique ID should consist out of things like a serial number or a hardware mac address of the device.

Reply:
pyvlx does not provide serial_number of the node, yet. I wrote a fix for pyvlx, [PR](https://github.com/Julius2342/pyvlx/pull/37) is created.
Shall I close this PR and reopen again, when pyvlx was updated?

Reply:
Yeah, that is generally a good practice. I save us looking at the PR every time 😉 

Great work btw! 👍

Reply:
@tschamm : I merged your PR within pyvlx, [taged a new version](https://github.com/Julius2342/pyvlx/releases/tag/0.2.14) and pushed it to [pypi](https://pypi.org/project/pyvlx/) !

Reply:
Also no need to add `DOMAIN`. Unique IDs are scoped within integration domain + platform